code/datums/components/orbiter.dm

```diff
@@ -136,7 +136,7 @@
 	for(var/atom/movable/movable_orbiter as anything in orbiter_list)
 		if(QDELETED(movable_orbiter) || movable_orbiter.loc == newturf)
 			continue
-		movable_orbiter.abstract_move(newturf)
+		movable_orbiter.forceMove(newturf)
```

Comment:
What is the difference between `abstract_move` and `forceMove`?

Reply:
**`forceMove`** uses `doMove` while **`abstract_move`** updates `loc` directly. Both call `Moved` but only the former also calls `Entered` and its associated signals which we do not want for observers, which is why their `forceMove` was overridden to use `abstract_move` so this change is purely for potential non observer orbiters like wisps.

Reply:
Than again I only changed it to be abstract before to fix the "Orbiters can't follow things through the transit z-level" bug and this would cause it to happen again for non observers, so now I may just leave it be until such time as it actually causes an issue.